docs/contribute.rst

```diff
@@ -0,0 +1,121 @@
+.. include:: ../.github/CONTRIBUTING.rst
+
+
+First-time setup
+~~~~~~~~~~~~~~~~
+
+- You need to install the third version of `Python`_.
+For example, Python3.7
+
+Then `to create and activate a virtual environment`_.
+And install `tox`_.
+
+- `Install git`_
+
+- `Configure git`_:
+
+1. Please, identify yourself::
+
+        $ git config --global user.name "firstname lastname"
```

Comment:
I think it would be better to have just 4-space indentation across docs:
```suggestion
    $ git config --global user.name "firstname lastname"
```